ocaml/xapi/xapi_vm_migrate.ml

```diff
@@ -153,6 +153,8 @@ let pool_migrate ~__context ~vm ~host ~options =
 		raise (Api_errors.Server_error (Api_errors.vm_failed_shutdown_ack, []))
 	| Xenops_interface.Cancelled _ ->
 		TaskHelper.raise_cancelled ~__context
+	| Xenops_interface.Storage_backend_error _ ->
+		raise (Api_errors.Server_error (Api_errors.sr_backend_failure, []))
```

Comment:
Don't we have a number we can at least put in? I see you're discarding the value from the `Storage_backend_error` constructor. How is this normally raised by the XenAPI?